README.md

```diff
@@ -0,0 +1,130 @@
+# .NET 9 SDK extension
+
+## How to use
+You need to add following lines to flatpak manifest:
+
+```json
+"sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet9"
+],
+"build-options": {
+    "append-path": "/usr/lib/sdk/dotnet9/bin",
+    "append-ld-library-path": "/usr/lib/sdk/dotnet9/lib",
+    "env": {
+        "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet9/lib/pkgconfig"
+    }
```

Comment:
use append-pkg-config-path or prepend-pkg-config-path. This is wrong in every dotnet extension.